learn/react-native/app-loader.md

```diff
@@ -47,17 +72,23 @@ In the following example, apply child Widgets of `widgetName` to show the Skelet
 
 #### Change Skeleton Colors
 
-You can control Skeleton Loader colors using styles. There are three style properties as following: 
+You can customise the color, width, and other properties of Skeleton Loader . These changes can be applied at three levels:
 
-1. skeleton
-2. skeleton-gradient
-3. skeleton-gradient-foreground
+- Page level
+- Widget level
+- Class level
+
+During Skeleton customization, Skeleton Loader is categorized into three classes where,
+
+1. skeleton: It is the outer container of the Skeleton component.
```

Comment:
Use apt formatting, such as code blocks for classes in this case. And use a header for this section.